src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -107,44 +108,69 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
             );
         }
 
-        public async Task<Document> CleanupAsync(Document document, CancellationToken cancellationToken)
+        public async Task<Document> CleanupAsync(
+            Document document, IProgressTracker progressTracker, CancellationToken cancellationToken)
         {
             var docOptions = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
 
+            // add one item for the 'Remove Usings', one for 'Sort using' and one item for the
+            // 'format' actions we'll do last
+            progressTracker.AddItems(3);
```

Comment:
I've fixed this.  I didn't want a "preparing to run cleanup" message because you'd see:

preparing...
applying whatever ...
preparing ...
applying whatever ...

Ad nauseum.  This would create a lot of unappealing visual clutter.

So i went with a different approach.  We now just use the names of the actual options as they appear in Tools|Options.  And we show that as the description for the entirety of that op.  i.e. we set the description, then we compute the diagnostics, and then we apply teh fix.  So the cost is accurately associated with each Option.

We also effectively update the description to the next thing we're doing after each step of the entire algorithm.